pkgs/development/python-modules/flask-appbuilder/default.nix

```diff
@@ -25,18 +27,30 @@
 
 buildPythonPackage rec {
   pname = "flask-appbuilder";
-  version = "3.3.4";
+  version = "4.1.3";
+  format = "setuptools";
+
+  disabled = pythonOlder "3.7";
 
   src = fetchPypi {
     pname = "Flask-AppBuilder";
     inherit version;
-    sha256 = "180swqkcp34vkarqb50dpn2300a6qdgihliqfflp0brab9v2n1hg";
+    sha256 = "sha256-8NaTr0RcnsVik/AB4g8QL+FkcRlgkkASFe8fXIvFt/A=";
   };
 
-  # See here: https://github.com/dpgaspar/Flask-AppBuilder/commit/7097a7b133f27c78d2b54d2a46e4a4c24478a066.patch
-  #           https://github.com/dpgaspar/Flask-AppBuilder/pull/1610
-  # The patch from the PR doesn't apply cleanly so I edited it manually.
-  patches = [ ./upgrade-to-flask_jwt_extended-4.patch ];
+  patches = [
+    (fetchpatch {
+      # https://github.com/dpgaspar/Flask-AppBuilder/pull/1734
+      name = "flask-appbuilder-wtf3.patch";
+      url = "https://github.com/dpgaspar/Flask-AppBuilder/commit/bccb3d719cd3ceb872fe74a9ab304d74664fbf43.patch";
+      sha256 = "sha256-24mlS3HIs77wKOlwdHah5oks31OOmCBHmcafZT2ITOc=";
+      excludes = [
+        "requirements.txt"
+        "setup.py"
+        "examples/employees/app/views.py"
+      ];
```

Comment:
I'm little bit worried regarding this patch. The PR was broken, so I've create another PR, which resolved the issues. Should we switch to the other patch? (In next release the patch is included)
PR: https://github.com/dpgaspar/Flask-AppBuilder/pull/1904
The squashed commit: https://github.com/dpgaspar/Flask-AppBuilder/commit/b010cdeae56d5db4a91192aa41b44bdc6efa699c

Reply:
I've noticed new versions are out, so the patch can be dropped entirely if we update
- Airflow to 2.4.2 https://github.com/apache/airflow/compare/2.4.1...2.4.2#diff-fa602a8a75dc9dcc92261bac5f533c2a85e34fcceaff63b3a3a81d9acde2fc52L111
- Flask-AppBuilder to 4.1.4 https://github.com/dpgaspar/Flask-AppBuilder/compare/v4.1.3...v4.1.4#diff-60f61ab7a8d1910d86d9fda2261620314edcae5894d5aaa236b821c7256badd7

But these changes would be better to get to master first probably.

Reply:
That sounds like a plan. If you haven't already done the team programming session, bumping to airflow 2.4.2 and applying the appbuilder patches would be an awesome contribution (both for us and upstream).